spec/requests/admin/roles_spec.rb

```diff
@@ -58,6 +60,43 @@
     end
   end
 
+  context 'when user has permissions to manage roles' do
+    let(:user_role) { Fabricate(:user_role, permissions: UserRole::FLAGS[:manage_users]) }
+
+    before { sign_in Fabricate(:user, role: user_role) }
+
+    context 'when target role permission outranks user' do
+      let(:role) { Fabricate :user_role, position: user_role.position + 1 }
```

Comment:
Why the style inconsistency?

Reply:
Sorry... in what sense?

Reply:
I meant regarding the use of parentheses. I kind of thought this was the first time we called `Fabricate` like that. It isn't, though, but there's 10 times as many `Fabricate(:…)` calls than `Fabricate :…` calls in the specs

Reply:
Oh I see ... updated to at least be consistent within this file and use parens.